data/presets/presets/place/welcome_sign.json

```diff
@@ -0,0 +1,15 @@
+{
+  "icon": "maki-lighthouse",
+  "geometry": [
+    "point",
+    "area"
+  ],
+  "terms": [
+    "new location"
+  ],
+  "tags": {
+    "description": "A sign welcoming users to a place",
+    "place": "welcome"
```

Comment:
Looks like the tags for this as specified in #7846 are `tourism=information`, `information=board`, and `board_type=welcome_sign`. These should go under the property `tags` instead of `description` and `place`.